File: CSIT111/Lecture_2/Lecture_Labs/Lecture_Notes_Examples/Converter.py
# A program that converts miles into kilometres
# input: distance in miles > output: distance in kilometres
class Converter:
    # --- fields ---
    disInmiles = 0.0
    disInKilometers = 0.0
    COEFFICIENT = 1.60935
    # --- definition of the method main() ---
    def main(self):
        # Create a scanner that can numbers, or words
        disInMiles = float(input("Enter the distance in miles: ")) # input a number typed by a user
        # calculate the distance
        disInKilometers = disInMiles * self.COEFFICIENT
        # output the result
        print(str(disInMiles) + " miles is " + str(disInKilometers) + " kilometres")

converter = Converter()
converter.main()

File: CSIT111/Lecture_2/Lecture_Labs/Lecture_Notes_Examples/Converter.java
// A program that converts miles into kilometres
// input: distance in miles > output: distance in kilometres
import java.util.Scanner;
public class Converter {
    // --- fields ---
    private static double disInMiles;
    private static double disInKilometers;
    private static final double COEFFICIENT = 1.60935;
    // --- definition of the method main() ---
    public static void main(String[] args) {
        // Create a scanner that can numbers, or words
        Scanner keyboard = new Scanner(System.in);
        System.out.print("Enter the distance in miles: ");
        disInMiles = keyboard.nextDouble(); // input a number typed by a user
        // calculate the distance
        disInKilometers = disInMiles * COEFFICIENT;
        // output the result
        System.out.println(disInMiles + " miles is " + disInKilometers + " kilometres");
        keyboard.close();
    }
}
